appserver/tests/appserv-tests/devtests/security/soteria/common/src/main/resources/arquillian.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://jboss.org/schema/arquillian" xsi:schemaLocation="http://jboss.org/schema/arquillian
+    http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+    <defaultProtocol type="Servlet 3.0"/>
+
+    <container qualifier="liberty">
```

Comment:
We need to replace it with glassfish specific configuration, since Arquillian needs to connect to a running instance of glassfish.

I tested with following and all 12 tests were passing.

```xml
<container qualifier="glassfish" default="true">
        <configuration>
            <property name="glassFishHome">$GLASSFISH_HOME</property>
            <property name="adminHost">localhost</property>
            <property name="adminPort">4848</property>
            <property name="allowConnectingToRunningServer">true</property>
        </configuration>
    </container>

    <container qualifier="glassfish-auth">
        <configuration>
            <property name="glassFishHome">$GLASSFISH_HOME</property>
            <property name="adminHost">localhost</property>
            <property name="adminPort">4848</property>
            <property name="adminUser">admin</property>
            <property name="allowConnectingToRunningServer">true</property>
            <!--property name="adminPassword">admin</property-->
        </configuration>
    </container>
```
Here $GLASSFISH_HOME is $INSTALLATION_DIRECTORY/glassfish5 unlike S1AS_HOME which also includes subdirectory glassfish within glassfish5.